src/components/gamepage/GamesPageImageGrid/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+// line 7-8-9 are mock datas
+import zelda from "../../../images/mockZeldaPoster.jpg";
+import gta from "../../../images/mockGTAPoster.jpg";
+import destiny from "../../../images/mockDestinyPoster.jpg";
+
+const mockDataArr = [
+  { name: "Grand theft auto", img: { src: gta, alt: "Grand theft auto" } },
+  { name: "Zelda", img: { src: zelda, alt: "Zelda" } },
+  { name: "Destiny", img: { src: destiny, alt: "Destiny" } },
+];
+
+const GamesPageImageGrid = () => {
+  const [t] = useTranslation();
+  const gameGridTitle = t("gamesPage.title");
```

Comment:
i would inline this if its not being used more than once

Reply:
PTAL